README.md

```diff
@@ -60,35 +60,35 @@ liota.conf provides path to find out various configuration & log files. When ini
 * Looks in the current working directory '.'
 * User's home directory '~'
 * A LIOTA_CONF environment variable
-* Finally the default location for every installation: /etc/liota/conf.
+* Finally the default location for every installation: /etc/liota/. (note this will need to be copied from the system doc directory, typically /usr/share/doc/liota-&lt;version&gt;/config/)
```

Comment:
If I get this change correct, user have to manually copy the required files from system doc directory to "/etc/liota"